nix/rust.nix

```diff
@@ -39,7 +39,8 @@
                   Security
                   SystemConfiguration
                 ] ++ [
-                  libiconv
+                  # error: don't yet have a `targetPackages.darwin.LibsystemCross for x86_64-apple-darwin`
```

Comment:
Can you elaborate on this comment? Is this an actual error message printed on Intel Mac?

Reply:
Yes, its an actual error message. I will add a short description above the error message.

Reply:
https://github.com/juspay/omnix/pull/158/commits/3593d1c488265f66a58e3d97c9b447b62c4ecdd1